src/PostgREST/Parsers.hs

```diff
@@ -171,7 +171,7 @@ pOrderTerm =
   <|> OrderTerm <$> pField <*> pure Nothing <*> pure Nothing
 
 pLogicTree :: Parser LogicTree
-pLogicTree = try (pNotLogicOp <*> (char '{' *> pLogicTree) <*> (char ',' *> pLogicTree <* char '}'))
+pLogicTree = try (pNotLogicOp <*> (char '(' *> pLogicTree) <*> (pDelimiter *> pLogicTree <* char ')'))
```

Comment:
Steve noted that the haskell url parse turns `+` into a space. And yeah I also find the dot separators to be hard to read. What about a semicolon?

```
/entities?or=(id.eq.1;id.eq.2;id.eq.3)&select=id,name
```